moviebot/nlu/annotation/joint_bert/slot_mapping.py

```diff
@@ -45,6 +45,12 @@ class JointBERTSlot(EnumWithMapping):
     I_PREFERENCE_KEYWORDS = auto()
     B_PREFERENCE_YEAR = auto()
     I_PREFERENCE_YEAR = auto()
+    B_PREFERENCE_TIME = auto()
+    I_PREFERENCE_TIME = auto()
+    B_PREFERENCE_COMPANION = auto()
+    I_PREFERENCE_COMPANION = auto()
+    B_PREFERENCE_LOCATION = auto()
+    I_PREFERENCE_LOCATION = auto()
```

Comment:
Add _CONTEXT_ in the name so we know what these refer to. E.g., `B_PREFERENCE_CONTEXT_TIME`